src/index.js

```diff
@@ -0,0 +1,35 @@
+import 'bootstrap';
+import 'jquery';
+import { updateforecast, updateweather } from './weather_view';
+import { fetchforecast, fetchweather } from './weather_api';
+
+const form = document.getElementById('form');
+const city = document.getElementById('city');
+
+form.addEventListener('submit', (e) => {
```

Comment:
You should use a toggle control to determine if the result will be shown in Celsius or Fahrenheit ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

Reply:
This comment was written after I asked for the second review, I tried my best to catch up on this mistake before someone checks my project, now I have fixed it, when you click on celsius it will turn to Fahrenheit.

Reply:
@AbdumurodovaZulfizar Sorry for the late comment, I actually wrote the comment for some reason it didn't appear in here. 🙏 
Thank you for your understanding.